frontend/www/js/omegaup/components/identity/Editv2.vue

```diff
@@ -103,69 +103,69 @@ import Autocomplete from '../Autocomplete.vue';
   },
 })
 export default class IdentityEdit extends Vue {
-  @Prop() identity!: omegaup.Identity;
+  @Prop({ default: null }) identity!: omegaup.Identity;
   @Prop() countries!: iso3166.Country[];
 
   T = T;
   typeahead = typeahead;
-  username = this.identity.username;
-  name = this.identity.name;
-  gender = this.identity.gender;
-  school = this.identity.school;
-  schoolId = this.identity.school_id;
-  selectedCountry = this.identity.country_id ?? 'MX';
-  selectedState = this.identity.state_id;
+  selectedIdentity = Object.assign(
+    <omegaup.Identity>{
+      username: '',
+      classname: '',
+      name: '',
+      gender: '',
+      school: '',
+      school_id: 0,
+      country_id: '',
```

Comment:
```suggestion
      country_id: 'MX',
```

así se preserva el default.

Reply:
Listo